tortoise/tracer.go

```diff
@@ -445,6 +441,46 @@ func (m *MalfeasanceTrace) Run(r *traceRunner) error {
 	return nil
 }
 
+type headerWithValidity struct {
+	Headers  types.BlockHeader `json:"header"`
+	Validity bool              `json:"validity"`
+}
```

Comment:
`{"validity": true`} looks weird. How about
```suggestion
type headerWithValidity struct {
	Header types.BlockHeader `json:"header"`
	Valid  bool              `json:"valid"`
}
```